plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2224,7 +2229,7 @@ void read_write::send_transaction(const read_write::send_transaction_params& par
                }
 
                const chain::transaction_id_type& id = trx_trace_ptr->id;
-               next(read_write::send_transaction_results{id, output});
+               next(read_write::send_transaction_results{id, expiration, output});
             } CATCH_AND_CALL(next);
```

Comment:
Adding the expiration here could be done in a separate PR so that you could have the info you need for your plugin.